packages/client/src/containers/Login.container/SignupForm.js

```diff
@@ -52,14 +62,18 @@ const SignupForm = () => {
       {formErrors.password && (
         <p className="form-erros">{formErrors.password}</p>
       )}
-
       <button
         onClick={handleSubmit}
         className="signup-submit-button"
         type="submit"
       >
         Sign up
-      </button>
+      </button>{' '}
+      {Object.keys(formErrors).length === 0 && isSubmit ? (
```

Comment:
this will be redundant once you refactor the handler
```suggestion
      { isSubmit ? (
```